### PR TITLE
All heads of staff now spawn with an energy pistol

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: All heads of staff, except for the HoS, now spawn with an energy pistol
/:cl:

Why: telebatons aren't that good of a self defense. Mini e-guns seem pretty balanced for this.

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
Deosnt precedent state that lethal weaponry stays within sec and hop/cap?

**Comment by MoreRobustThanYou** [comment]
TECHNICALLY, the telebaton can be a lethal weapon. But that's honestly shit self defense, it's more for getting the clown out.

File: code/modules/jobs/job_types/security.dm [hunk 1]
```diff
@@ -51,7 +51,7 @@ Head of Security
 	suit_store = /obj/item/gun/energy/e_gun
 	r_pocket = /obj/item/device/assembly/flash/handheld
 	l_pocket = /obj/item/restraints/handcuffs
-	backpack_contents = list(/obj/item/melee/baton/loaded=1)
+	backpack_contents = list(/obj/item/melee/baton/loaded=1, /obj/item/gun/energy/e_gun/mini = 1)
```
Comment (MMMiracles) [inline code/modules/jobs/job_types/security.dm]: giving the man with full access to the armory, a spare energy gun, and his own personalized weapon another gun.

Comment (ExcessiveUseOfCobblestone) [reply code/modules/jobs/job_types/security.dm]: he should only have the sword the nobility don't use poor man's weaponry.

**Comment by ExcessiveUseOfCobblestone** [comment]
Also let's not give heads ranged stuns tyvm

**Comment by MrDoomBringer** [comment]
@MoreRobustThanYou technically fists are a lethal weapon too, i meant ranged lethals i guess

**Comment by ma44** [comment]
Git Gud

**Comment by Spaceinaba** [comment]
as a head of staff player I approve of this change

**Comment by MoreRobustThanYou** [comment]
Lore-wise, it'd make sense for NanoTrasen to equip it's heads of staff with basic self-defense weapons. Can't be having them god darn syndicates stealing the ~phoron~plasma!

**Comment by praisenarsie** [comment]
@MoreRobustThanYou this is /tg/, not bay

**Comment by kevinz000** [comment]
this is tg not hippie

**Comment by Spaceinaba** [comment]
this is tg not citadel

**Comment by Slignerd** [comment]
Jesus christ.

I mean, now that I said this, I can't honestly say whether I agree with this PR or not. But an energy gun seems like a bit of an overkill.

Make it a hybrid taser instead. CMO, CE and RD really don't need an e-gun. Also, if you want to do this, add rechargers to head of staff offices.

**Comment by Slignerd** [comment]
Also, if you want to add the energy gun to the Captain's and HoP's loadout, you should remove it from their lockers.

**Comment by Spaceinaba** [comment]
what if they got a disabler pistol

**Comment by Slignerd** [comment]
Yeah, thinking back, a disabler sounds like a good option.

**Comment by Anonmare** [comment]
A disabler pistol on high-pop sounds reasonable-r. Telebatons are fine enough for controlling low/medium pop departments

**Pull request closed without merging**